applications/inscription_indexer.md

```diff
@@ -0,0 +1,174 @@
+| **0a.** | License                  | Apache 2.0                                                                                                                                                                                                                                    |
+| **0b.** | Documentation            | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                               |
+| **0d.** | Docker                   | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.                                                                                                                                 |
+|      1. | dot-20  Standard           | Writing the DOT-20 Protocol Standard                                                                                                                                                                                                          |
+|      2. | dot-20                   | Implementation of the DOT-20 Protocol: Python Package                                                                                                                                                                                         |
+|      3. | crawler                  | A Python package for fetching DOT-20 related transactions on Substrate chain                                                                                                                                                                  |
+|      4. | db                       | A Python package for the data table structure and related APIs involved in the indexer                                                                                                                                                                                                              |
+|      5. | indexer                  | The core code of the indexer, integrating various components to complete the entire functionality of the indexer.                                                                                                                                                                                                                      |
+
+### Milestone 2 - Storing User Tick Balances with Sparse Merkle Trees
```

Comment:
> What are "user tick balances" and why do they need storing in a MySQL database?

It is the balance of the user's token. In Inscription, we prefer to call it tick. Because in MySQL, it brings difficulties to reconciliation, it needs to be stored in the Merkle tree structure. When different indexers need to know whether they are the same, they only need to compare the Merkle roots.